apps/storefront/src/utils/b3Login.ts

```diff
@@ -7,6 +7,13 @@ export const loginJump = (navigate: NavigateFunction, isClearSession = false) =>
     global: { loginLandingLocation, recordOpenHash, setOpenPageFn },
   } = store.getState();
   if (loginLandingLocation === '1' && !recordOpenHash) {
+    const { href } = window.location;
+    if (href.includes('login.php')) {
```

Comment:
```suggestion
    if (window.location.includes('login.php')) {
```

Reply:
Maybe you mean to change it to:
"window.location.href.includes('login.php') " ?

Reply:
yep, sorry, basically don't use the interim variable if it is only used once